src/common/components/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import Select, { Option } from '../Select/Select'
+import cn from 'classnames'
+
+export type PageHeaderRoutes = {
+  label: string
+  route?: string // in the future when clicking the link it should change current route...
```

Comment:
we are good to remove this comment. :)

Reply:
@CrazyOptimist we are all team, we work together, so good comments would be helpful for everyone to understand codes.